### ISSUE TITLE
Pass the hostname when requesting the DHCP lease

### ISSUE THREAD

**Issue by peerst** [grisp/grisp#28]
Would show up nicer on lists of DHCP servers and some DHCP serves (the TP-Link we use for demoing e.g) would not show clients which don't do this at all.

**Comment by sylane** [comment]
@ziopio  Do you know if what you tested in your DHCP-related PR, is passes the hostname to the DHCP server ?

**Comment by ziopio** [comment]
No, I need to test that, is it the reason why you had `domain grisp.local` in `resolv.conf`?

**Comment by ziopio** [comment]
The hostname is set in the erl_main.c code. With my DHCP PR I can spot the board hostname in my router ARP table and in the DHCP lease table.
